interfaces/api/v1/core/settings.py

```
@@ -173,6 +183,129 @@ def update_embedding_config(body: EmbeddingConfigUpdate):
+    """测试 LLM 配置是否能成功连接"""
+    try:
+        if body.provider == "anthropic":
+            success = await _test_anthropic_config(body.api_key, body.base_url)
+        else:
+            success = await _test_openai_config(body.api_key, body.base_url)
+
+        if success:
+            return TestConfigResponse(success=True, message="连接成功")
+        else:
+            return TestConfigResponse(success=False, message="连接失败")
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)